src/main/charts/bitbucket/templates/ingress.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.bitbucket.ingress.create }}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
```

Comment:
The content of this template describes a bog-standard Kubernetes Ingress. On its own, this isn't enough to work, the customer will need to provide the annotations that give the Ingress its flavour, e.g. for the nginx ingress controller.